Core/src/org/sleuthkit/autopsy/coreutils/AppDBParserHelper.java

```diff
@@ -0,0 +1,1321 @@
+     * @param toAddressList callee address list, may be empty
+     * @param startDateTime start date/time
+     * @param endDateTime end date/time
+     * 
+     * @return call log artifact
+     */
+    public BlackboardArtifact addCalllog(CommunicationDirection direction, 
+                                Account.Address fromAddress, 
+                                Collection<Account.Address> toAddressList, 
+                                long startDateTime, long endDateTime) {
+    
+         return addCalllog(direction, fromAddress, toAddressList, 
+                            startDateTime, endDateTime, 
+                            CallMediaType.UNKNOWN);
+    }
+    
+     /**
+     * Adds a TSK_CALLLOG artifact. 
+     * 
+     * Also creates an account instance for the caller/callees, 
+     * and creates a relationship between the device owner account and the caller account
+     * as well between the device owner account and each callee account
+     * 
+     * @param direction call direction
+     * @param fromAddress caller address, may be empty
+     * @param toAddressList callee address list, may be empty
+     * @param startDateTime start date/time
+     * @param endDateTime end date/time
+     * @param mediaType call media type
+     * 
+     * @return call log artifact
+     */
+    public BlackboardArtifact addCalllog(CommunicationDirection direction, 
+                                Account.Address fromAddress, 
+                                Collection<Account.Address> toAddressList, 
+                                long startDateTime, long endDateTime,
+                                CallMediaType mediaType) {
+    
+         return addCalllog(direction, fromAddress, toAddressList, 
+                            startDateTime, endDateTime, 
+                            mediaType,
+                            Collections.<BlackboardAttribute>emptyList());
+    }
+    
+    /**
+     * Adds a TSK_CALLLOG artifact. 
+     * 
+     * Also creates an account instance for the caller/callees, 
+     * and creates a relationship between the device owner account and the caller account
+     * as well between the device owner account and each callee account
+     * 
+     * @param direction call direction
+     * @param fromAddress caller address, may be empty
+     * @param toAddressList callee address list, may be empty
+     * @param startDateTime start date/time
+     * @param endDateTime end date/time
+     * @param mediaType called media type
+     * @param otherAttributesList other attributes
+     * 
+     * @return calllog artifact
+     */
+    public BlackboardArtifact addCalllog(CommunicationDirection direction, 
+                                Account.Address fromAddress, 
+                                Collection<Account.Address> toAddressList, 
+                                long startDateTime, long endDateTime,
+                                CallMediaType mediaType,
```

Comment:
You don't actually do anything with CallMediaType

Reply:
Yes, we don't currently have any attribute to show the call type. I added it to the interface now for completeness, so we wont have to alter the interface when we add the attribute later.